lib/dogma/util/script_strings.ex

```diff
@@ -7,98 +7,106 @@ defmodule Dogma.Util.ScriptStrings do
   for Elixir code and reporting errors when there should be none.
   """
 
+  @sigil_delimiters [{"|", "|"}, {~S("), ~S(")}, {"'", "'"}, {"(", ")"},
+                      {"[", "]"}, {"{", "}"}, {"<", ">"}]
+  @all_string_sigils @sigil_delimiters
+                      |> Enum.flat_map(fn({b, e}) ->
+                          [{"~s#{b}", e}, {"~S#{b}", e}]
```

Comment:
I think it would be more useful to remove all sigils, including custom ones. We can rename this module, change docs, etc. If we don't, we still have the same problems with not being able to tell if "comments" are inside sigils.

Anyone got a good name?